### ISSUE TITLE
RegressionTables in Pluto.jl notebooks

### ISSUE THREAD

**Issue by nilshg** [jmboehm/RegressionTables.jl#80]
Apologies if this has been discussed before, it seems very basic but I couldn't find anything in the closed issues: 

Would it be possible to return the output from a call to `regtable`? Currently it's just written to console or file, but when e.g. using the package in a Pluto notebook where the REPL is separate from the notebook it would be nice to have the ability to get the ASCII/HTML output and work with it in the notebook.

**Comment by jmboehm** [comment]
Looks like a duplicate of #7 ?

**Comment by nilshg** [comment]
Ah yes I missed that, sorry!

**Comment by greimel** [comment]
for future readers, here is something that works.

```julia
function regtable_html(args...; kwargs...)
    io = IOBuffer()
    regtable(args...; kwargs...,
        renderSettings=RegressionTables.htmlOutput(),
        out_buffer = io)
    String(take!(copy(io))) |> HTML
end
```

it would be nice, if this would somehow automatically work.

**Comment by jmboehm** [comment]
The [new solution](https://jmboehm.github.io/regtables-pluto.jl.html) implemented in #125 is a nice way of doing this.
